src/jit/lowerxarch.cpp

```diff
@@ -2754,14 +2754,14 @@ void Lowering::TreeNodeInfoInitSIMD(GenTree* tree)
             else
             {
 
-                // Need two SIMD registers as scratch.
+                // Need one SIMD registers as scratch.
```

Comment:
On line 2759 too.


---
In reply to: [89228050](https://github.com/dotnet/coreclr/pull/8229#discussion_r89228050) [](ancestors = 89228050)